src/revocation.rs

```diff
@@ -118,8 +237,7 @@ pub(crate) async fn run_revocation_service() -> Result<()> {
                     "Revocation signature validated for revocation: {}",
                     msg_payload
                 );
-                // TODO: Implement callback
-                //callback(msg_payload)
+                let _ = run_revocation_actions(Some(msg_payload))?;
```

Comment:
@ashcrow What was the idea behind passing `msg_payload` to this function in your previous commit? I've done this, but I don't end up using it. Is there something we should be parsing from the payload?

Reply:
It comes from https://github.com/keylime/keylime/blob/master/keylime/revocation_notifier.py#L122 which leads to [this](https://github.com/keylime/keylime/blob/1d0f695c0848b001acb0ab01de46094494884506/keylime/keylime_agent.py#L630) and more importantly [this execution](https://github.com/keylime/keylime/blob/1d0f695c0848b001acb0ab01de46094494884506/keylime/keylime_agent.py#L592) where it finally runs and passes the content into the action. Looking at [this revocation action](https://github.com/keylime/keylime/blob/1d0f695c0848b001acb0ab01de46094494884506/keylime/revocation_actions/update_crl.py#L22) it does seem like it's expected to pass the data through.

Reply:
In the python code it actually imports the code and calls the method directly. I think we will need to update the actions to be able to take the payload through `stdin` or something similar.

@mpeters @lukehinds @puiterwijk thoughts?

Reply:
@ashcrow Thanks for looking into that. Yeah, it looks like in `execute()` it just checks that it's a revocation and then does nothing else with it?

One thing this PR hasn't dealt with, though, is the CRL, if we are expecting to do that.